specification/communication/data-plane/Microsoft.CommunicationServicesSms/stable/2021-03-07/examples/send.json

```diff
@@ -0,0 +1,56 @@
+{
+  "parameters": {
+    "SendMessageRequest": {
+      "from": "+18001110000",
+      "to": [
+        "+11234567890",
+        "+11234567891"
+      ],
+      "message": "Hello world!",
+      "sendSmsOptions": {
+        "enableDeliveryReport": true
+      }
+    },
+    "api-version": "2021-03-07",
+    "endpoint": "https://contoso.communication.azure.com/"
+  },
+  "responses": {
+    "202": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "to": "+11234567890",
+            "messageId": "Outgoing_20200610203725bfd4ba70-70bf-4f77-925d-c0bdb5161bb3",
+            "httpstatuscode": 202
+          },
+          {
+            "to": "+11234567891",
+            "messageId": "Outgoing_2020061020025100d1-d07e-4b19-b25d-13525d552822",
+            "httpstatuscode": 202
+          }
+        ],
+        "nextLink": "null"
```

Comment:
When nextLink is null, it shouldn't be included as a property at all. We can change the example here with a fake nextLink string, something like `"/someNextLink"`